flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/RestfulGateway.java

```diff
@@ -123,4 +126,19 @@
 			@RpcTimeout Time timeout) {
 		throw new UnsupportedOperationException();
 	}
+
+	/**
+	 * Requests the statistics on operator back pressure.
+	 *
+	 * @param jobId       Job for which the stats are requested.
+	 * @param jobVertexId JobVertex for which the stats are requested.
+	 * @return A Future to the {@link OperatorBackPressureStats} or {@code null} if the stats are
```

Comment:
nit: This should be "A future to the {@link OperatorBackPressureStatsResponse}"?